src/Renderer.tsx

```diff
@@ -38,24 +40,38 @@ type Fallback = 'ignore' | 'warning' | 'passthru' | ComponentType<{ node: Node }
 
 interface Props<N extends Node | Node[]> {
     children?: ReactNode;
+    coverageEntries?: Record<number, CoverageEntry>;
     defaultComponents?: boolean;
     defaultFallback?: Fallback;
     nodes: N;
+    renderDate?: (date: string) => ReactNode;
```

Comment:
In the editor, we've been formatting the date by using `moment` and providing the desired date format.

However in the themes, we're using a custom component to render the date so I wanted to make it flexible.